### DISCUSSION TITLE
A drawing canvas app using React Native Skia!

### DISCUSSION THREAD

**Discussion by wobsoriano** [steveruizok/perfect-freehand#46] [Show and tell]
I was able to use perfect-freehand with [RN Skia](https://shopify.github.io/react-native-skia). It's so mooth!

https://github.com/wobsoriano/rn-perfect-sketch-canvas

**Comment by steveruizok** [comment]
Looks great!

**Reply by steveruizok** [reply]
You might also expose options like thinning, there are a few others in the lib.
